### ISSUE TITLE
NTR: stator components of bacterial flagellum

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#14283]
A reviewer of the cilia GO ontology manuscript has requested an additional component term as part of the bacterial flagellum. @paolaroncaglia,  @JohnvanDam, and I need to work on this promptly to meet October 30 deadline to submit revisions. I am also tagging @jimhu-tamu.

> 1: in the collection of bacterial flagellar components the authors could include the 'stator' components that form the stationary part of the flagellar engine. See. e.g.: 10.1002/9780470015902.a0000744.pub4. It should be quite straightforward to annotate the corresponding proteins that form this part of the flagella.

I don't have access to the DOI he cites = http://onlinelibrary.wiley.com/doi/10.1002/9780470015902.a0000744.pub4/abstract? 

However, this more recent review seems really good:
**Zhao X, Norris SJ, Liu J. Molecular architecture of the bacterial flagellar
motor in cells. Biochemistry. 2014 Jul 15;53(27):4323-33. PMID: 24697492**

Here are couple quotes from it below which are very informative as to appropriate course of action:

> Bacterial Flagellar Motor
> The structure and function of bacterial flagella have been extensively studied in model systems Salmonella enterica and Escherichia coli, as summarized in several recent comprehensive reviews.4,11−14 Of approximately 50 genes involved in the expression and assembly of the flagellum, ∼20 produce integral flagellar components. The flagellum consists of the motor, the hook, and the helically shaped flagellar filament (Figure ​(Figure1A,B).1A,B). The flagellar motor can be divided into several morphological domains (Figure ​(Figure1C,D):1C,D): the MS ring (FliF, the base for the flagellar motor), the C ring (FliG, FliM, and FliN, the switch complex regulating motor rotation), the export apparatus (a large complex exporting flagellar substrates), the rod (connecting the MS ring and the hook), the L and P rings on the rod (thought to serve as bushings at the outer membrane and at the peptidoglycan layer, respectively), and the stator, which is the motor force generator embedded in the cytoplasmic membrane.
> 

Taking the above paragraph and converting it into outline form, this review clearly supports including the 'stator' as part_of the bacterial flagellum:

The flagellum consists of 
- the motor
- the hook 
- the helically shaped flagellar filament

The flagellar motor can be divided into several morphological domains: 
- the MS ring (FliF, the base for the flagellar motor)
- the C ring (FliG, FliM, and FliN, the switch complex regulating motor rotation)
- the export apparatus (a large complex exporting flagellar substrates)
- the rod (connecting the MS ring and the hook)
- the L and P rings on the rod
- the stator, which is the motor force generator embedded in the cytoplasmic membrane.

Looking at the existing child terms of 'bacterial-type flagellum part', I see the term 'bacterial-type flagellum basal body' and was wondering what that was. Another quote from the same paper provides the answer:

> Cryo-electron microscopy (cryo-EM) studies have provided the most detailed structures of the purified basal body, which contains the MS ring, the C ring, the rod, and the P and L rings.43 However, these structures do not contain components of the stator and the export apparatus.

**Comment by krchristie** [comment]
The review I cited previously has a great diagram with an informative caption:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4221660/figure/fig1/

> Schematic models of the external flagellum of E. coli (A and C) and periplasmic flagella of B. burgdorferi (B and D). Periplasmic flagella are distinct from the external flagella, as they are enclosed within the outer membrane and their flagellar motors are considerably larger and more complex. However, the core architecture of the two flagellar types is comparable. Shared structures include the MS ring, the C ring, the rod, the hook, the filament, the stator, and the export apparatus.

Based on this review, I propose to add a couple new terms and change the parentage of one existing term under the existing term:

**bacterial-type flagellum part**
-- bacterial-type flagellum motor
--- bacterial-type flagellum basal body [existing term, but **move to be part_of new motor term**]
--- bacterial-type flagellum stator complex [**GO:new**]
--- bacterial-type flagellum export apparatus [**GO:new**]

Looks like I'll also need to modify the existing definition of **bacterial-type flagellum basal body** 

_One of the three major substructures of the flagellin-based flagellum; a structure consisting of a rod, a series of rings, the Mot proteins, the switch complex and the_ **flagellum-specific export apparatus**. _The rings anchor the flagellum to the cytoplasmic membrane (MS ring), the peptidoglycan (P ring) and the outer membrane (L ring). Examples of this component are found in bacteria._

since it currently mentions the export apparatus (though there is no term for this). I plan to base my new definition on the quote I previously included that starts with "Cryo-electron microscopy...".

**Comment by paolaroncaglia** [comment]
@krchristie
Many thanks. I'm still at the GOC meeting so haven't had a chance to review all of this yet, but I've found the full text of the original DOI the reviewer quoted, in case it's helpful. Attaching it here.

**Comment by paolaroncaglia** [comment]
/Users/paola/Desktop/a0000744.pdf

**Comment by paolaroncaglia** [comment]
@krchristie
I looked at the DOI provided by the reviewer, and at your comments above, and I agree with your suggested course of action. Two more suggestions:
- The DOI says "…The stator units, also called torque-generating units…" so I'm wondering if we should include the string "torque" in a synonym;
- Someone should annotate the stator components to the new stator term. @jimhu-tamu please?

**Comment by paolaroncaglia** [comment]
Also mentioning @suzialeksander in case she could help with the annotations please. Thanks.

**Comment by suzialeksander** [comment]
Yes, I think I've already found a paper acceptable for 'stator'.  We can do this as soon as there's a term.

**Comment by krchristie** [comment]
I've added the three terms and made some changes to the relationships for the 'bacterial-type flagellum basal body' term. For the stator term, I added a synonym with 'torque generator' as Paola suggested, as well as a similar one with 'motor force generator'.

I've just realized I still need to modify the def of 'bacterial-type flagellum basal body', so I'll do that shortly.

**Comment by jimhu-tamu** [comment]
Now that this is reopened... we can try to get our local bacterial motor expert to look at this set of terms. cc: @dsiegele

**Comment by krchristie** [comment]
@paolaroncaglia supported my proposal and @suzialeksander said she had a paper appropriate for annotation of stator proteins once the terms were available, so I've gone ahead and made the new terms so Suzi could annotate sooner rather than later. 

But I would definitely love it if @dsiegele would take a look and provide feedback, especially if anything needs to be changed.

Thanks!

**Comment by paolaroncaglia** [comment]
@krchristie I just noticed a tiny typo in the def below, could you please remove the extra "t"? Thanks!
id: GO:0120100
name: bacterial-type flagellum motor
def: "Tthe core architecture..."

**Comment by paolaroncaglia** [comment]
Hi @dsiegele,

Following up on @jimhu-tamu’s message above, we were wondering if you could please confirm if the revised ontology structure under 'bacterial-type flagellum part’ looks correct, in particular the new terms 
- bacterial-type flagellum motor
- bacterial-type flagellum stator complex
- bacterial-type flagellum export apparatus
and the revised term
- bacterial-type flagellum basal body [this was an existing term, but it has now been moved to be part_of the new bacterial-type flagellum motor term]

All terms above are viewable in AmiGO or QuickGO. Any question or concern please let us know. This is needed for a response to a paper submission, so your input would be much appreciated to confirm that we’ve done this correctly. (But if your availability is limited at this time, please let us know as we need to meet the journal’s deadline for re-submission.)

@suzialeksander has kindly offered to annotate to the new ‘stator’ term, as requested by a reviewer.  

Many thanks to you both!

Paola and Karen

**Comment by paolaroncaglia** [comment]
Hi @suzialeksander, 

As agreed, could you please annotate to the new term ‘bacterial-type flagellum stator complex’, to help us address this reviewer’s request:

“in the collection of bacterial flagellar components the authors could include the 'stator' components that form the stationary part of the flagellar engine. See. e.g.: 10.1002/9780470015902.a0000744.pub4. It should be quite straightforward to annotate the corresponding proteins that form this part of the flagella.”

We are fairly confident that the new term is correct and ok to use. Once the annotations are ready, it will be sufficient to say when they will be released - no need to push a release if it’s just for a very small number of annotations.

Many thanks for your help!

**Comment by krchristie** [comment]
I'm quite confident that these two terms are fine:
- bacterial-type flagellum stator complex
- bacterial-type flagellum export apparatus

My biggest question for @dsiegele would be if she agrees with the way I've made the basal body part_of the flagellum motor, as shown here:

**-- bacterial-type flagellum motor**
--- bacterial-type flagellum stator complex
--- bacterial-type flagellum export apparatus
**--- bacterial-type flagellum basal body** 
---- several existing terms that were already under flagellum basal body

**Comment by suzialeksander** [comment]
Hi, 
I tried to make an annotation yesterday to motB, and was making one today to motA (I'm working on protein2go so you shouldn't need to wait for a release).

However,  'GO:0120101 bacterial-type flagellum stator complex' is currently a CC.  The synonyms 'bacterial-type flagellum torque generator' and 'bacterial-type flagellum motor force generator' appear to me as Functions- should they both be appended with 'complex'? 

On a side note is there an existing Function term that could be used for these proteins? (Eg, 'bacterial-type flagellum torque generator')? Looks like they really only have Process terms like " GO:0071973 bacterial-type flagellum-dependent cell motility"

I know you're on a deadline so if the synonym issue at least could be addressed that would be great. 

Thanks
Suzi

**Comment by krchristie** [comment]
Hi  @suzialeksander 

I put in those synonyms for the 'bacterial-type flagellum stator complex' because they were used in the cited reviews as synonyms for that structure, so it seemed appropriate to @paolaroncaglia and I to put them in as synonyms. If you feel it is appropriate, I can remove them, or I could add the word 'complex' to each of the synonyms to make it clear we are still talking about the complex here. Let me know which you prefer.

We did not address any MF terms. Our primary goal was the requested 'bacterial-type flagellum stator complex' term, but once I read a review it seemed that we should also add a CC term for the 'export apparatus'. Once we were adding these two CC terms that are not described as being part of the basal body, but still being part of the motor, it seemed appropriate to add the 'bacterial-type flagellum motor' term for organization. If you would like to have any MF terms, please open a new ticket with a proposal for what the term name should be, e.g. term name, definition, etc.

Annotations made in P2GO are only 'released' to public view about every 6-8 weeks. I will be making the ones for the requested eukaryotic term 'centriolar subdistal appendage' using P2GO also, so all of these annotations will likely make the same release.

**Comment by suzialeksander** [comment]
Hi,

We'd be satisfied if the CC synonyms were appended with "complex".

After talking with @jimhu-tamu, we do think this term should have the parentage "is_a" membrane, probably  'GO:0005887 integral component of plasma membrane' - there's a schematic in PMID:7892209  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC42404/pdf/pnas01484-0186.pdf.  This reflects the language in the definition.

If I'm reading it correctly it currently is 'part_of' the flagellum, which we agree with.

Also, we're not incredibly keen of including the protein names in the definition.  I think 'MotA/B' is not universal, as PMID:25197056 http://www.pnas.org/content/111/37/13523.full.pdf mentions "The stator consists of MotA and MotB for the H+ -driven motor of Escherichia coli and Salmonella enterica, and PomA and PomB for the Na+ -driven motor of Vibrio and Shewanella spp."  Can that reference to MotA and MotB be removed?

I've made a few annotations (IDA and IMP) to each MotA and MotB, so I'll try to do more later BUT there should be some for @paolaroncaglia to reference now.

Thank you!
Suzi

**Comment by krchristie** [comment]
Thanks @suzialeksander and @jimhu-tamu for the feedback! I just wanted to suggest that instead of removing the specific protein info, we could add more info to make it clear that the term is OK for both types of motors.

- [ ] **synonyms**: 
I will add the word 'complex' to the two synonyms.

- [ ] **definitions**:
Regarding the mentions to specific proteins. We often find it helpful to curators in identification of the correct term to have the mentions of specific proteins, but to put them in with more detail about which organism they come from. So, I could just remove them. Alternatively, I could change it from: 

> Def: A transmembrane complex embedded in the cytoplasmic membrane which is the motor force, or torque, generator of the bacterial-type flagellum. A ring of stator complexes (each comprised of MotA & MotB) assemble in the cytoplasmic membrane around the MS ring.

to something like:
> Def: A transmembrane complex embedded in the cytoplasmic membrane which is the motor force, or torque, generator of the bacterial-type flagellum. A ring of stator complexes assemble in the cytoplasmic membrane around the MS ring. The stator consists of MotA and MotB for the H+ -driven motor of Escherichia coli and Salmonella enterica, and PomA and PomB for the Na+ -driven motor of Vibrio and Shewanella species.

Let me know your preference.

- [ ] **parentage**
add parentage "is_a" 'GO:0005887 integral component of plasma membrane'

**Comment by dsiegele** [comment]
I apologize for taking so long to respond and join the discussion.

After reading other papers from several groups that work on bacterial flagella, think that the way that Zhao et al. (Biochemistry 2014) describe the organization of the flagellum (motor, hook, and filament) is not consistent with the field. It seems to be much more common to divide the bacterial flagellum into the basal body, the hook, and the helical filament (PMC:4869327, PMID: 10572114, PMID:12624192, xx). 

i) The **basal body** is embedded in the cell envelope (the plasma membrane, peptidoglycan cell wall, and, if one is present, the outer membrane). As well as anchoring the flagellum in the cell envelope, the basal body houses the secretion apparatus for export of the more distal components  of the flagellum and provides the power for flagellar rotation (the flagellar motor, which consists of the rotor and surrounding stators).

     _In Karen's first post there is a statement from Zhao et al. "Cryo-electron microscopy (cryo-EM)    
     studies have provided the most detailed structures of the  purified basal body, which    
     contains the MS ring, the C ring, the rod, and the P and L rings (ref 43). However, these 
    structures do not contain components of the stator and the export apparatus."_   In other 
     papers, the stator and the export apparatus are considered part of the
     basal body. My reading of the quoted statement is that the stator and export apparatus were
    just missing from the basal body preparations examined in ref. 43. 

ii) The **hook** is a flexible universal joint connected to the basal body that changes the angle of rotation to the filament.

iii) The **filament** functions as a propeller.

The **flagellar motor** consists of a _rotor_ surrounded by one or more _stators_.

The **rotor**, which is part of the basal body, has two components: i) a membrane anchor and ii) the motor switch complex, which controls the direction of flagellar rotation. 

The membrane anchor has been referred to as the MS ring in E. coli and Salmonella enterica type Typhimurium and as the M-ring or basal body ring in B. subtilis. 

My suggested organization of terms and suggested definitions: 

**bacterial-type flagellum part**  
--- bacterial-type flagellum basal body [existing term, <s>but move to be part_of new motor term</s>]
------- bacterial-type flagellum motor [GO:new]
------------ bacterial-type flagellum stator complex [GO:new]
------------ bacterial-type flagellum rotor complex 
------- bacterial-type flagellum secretion apparatus 


GO:0044461 ! bacterial-type flagellum part
def: Any constituent part of flagellum, a 20 nm diameter filament composed of subunits of flagellin driven passively at its base by a motor powered by <s>the transmembrane proton potential</> a transmembrane ion potential, typical a proton or sodium potential. Examples of this component are found in bacterial species.

GO:0009425 ! bacterial-type flagellum basal body
def: <s>The core of the motor, one of the three major substructures of the bacterial-type flagellin-based flagellum, the basal body contains the MS ring, the C ring, and the rod with one or more associated rings (P and sometimes also L). </s> Source: GOC:cilia, GOC:mtg_sensu, PMID:12624192, PMID:10572114, GOC:krc, PMID:24697492

def: One of the three major substructures of the bacterial-type flagellum, the basal body is embedded in the cell envelope (the plasma membrane, peptidoglycan cell wall, and, if one is present, the outer membrane); it houses the secretion apparatus that exports the more distal components and the flagellar motor. 

GO:0120100 ! bacterial-type flagellum motor
 def. A transmembrane complex embedded in the cytoplasmic membrane which is the motor force, or torque, generator of the bacterial-type flagellum. The motor consists of a membrane-anchored rotor complex surrounded by one or more stator complexes in the cytoplasmic membrane. The stator consists of a hetero-hexameric complex of 2 membrane proteins, A and B, with stoichiometry A4B2. Examples are the H+ -driven MotA-MotB stator complex of Escherichia coli and Salmonella enterica, and the Na+ -driven PomA-PomB stator complex of Vibrio and Shewanella species. The rotor complex consists of a membrane-anchored ring and the motor switch complex, which controls the direction of flagellar rotation. 


GO:0120101 bacterial-type flagellum stator complex
def: A hetero-hexameric complex of 2 membrane proteins, A and B, with stoichiometry A4B2. The A and B proteins form a channel through which the ions that power the flagellum flow. They form the stator, or nonrotating portion, of the flagellum motor with the B protein apparently attached to the peptidoglycan cell wall. (PMID: 12624192) Examples are the H+ -driven MotA-MotB stator complex of Escherichia coli and Salmonella enterica, and the Na+ -driven PomA-PomB stator complex of Vibrio and Shewanella species. 

bacterial-type flagellum rotor complex [GO:new]
def.   The rotor complex consists of a membrane-anchored ring and the motor switch complex, which participates in the conversion of proton/Na+ energy into the mechanical work of rotation and controls the direction of flagellar rotation.

**Comment by paolaroncaglia** [comment]
@suzialeksander @dsiegele @jimhu-tamu @krchristie
Thank you all so much for your feedback. 

@krchristie, I’ll leave the remaining edits to you, with many thanks in advance.

@suzialeksander, I don’t have access to P2GO, so could you please list here the annotations you made so far? So I can add them in our response letter. Thanks!

**Comment by suzialeksander** [comment]
Hi @paolaroncaglia,

These are the annotations I've made.  If you needed them in a GAF format or anything let me know. 

P09348/E coli motA:
GO:0120101, IMP to PMID:3050128
GO:0120101, IDA to PMID:14705928

P0AF06/E coli motB:
GO:0120101, IMP to PMID:3050128
GO:0120101, IDA to PMID:14705928
GO:0120101, IMP to PMID:6374467

O06873/Vibrio alginolyticus pomA
GO:0120101, IDA to PMID:19894756
GO:0120101, IMP to PMID:24398784

O06874/Vibrio alginolyticus pomB
GO:0120101, IDA to PMID: 19894756
GO:0120101, IMP to PMID:21602350

**Comment by paolaroncaglia** [comment]
@suzialeksander
Thanks, that's all I need! Have a great day.

**Comment by krchristie** [comment]
I think I have made all the requested changes, to synonyms & parentage specifically of the stator term requested by @suzialeksander & @jimhu-tamu, and also all the def changes, 'part_of' relationship changes, and the new term @dsiegele suggested. I've attached the three references she cited for all of the terms I changed the definitions of. The colorized diff shows the changes really clearly: https://github.com/geneontology/go-ontology/commit/792543edf8b2467e61fd64301dcd4ef833b928ed

Let me know if you see anything I missed.

**Comment by paolaroncaglia** [comment]
@krchristie
Thank you so much for all your work! I updated the response letter to include all info.
Now, this is really icing on the cake, but may I please suggest a few minor changes in the definitions of the terms below - if you agree and if and when you have time:

id: GO:0044461
name: bacterial-type flagellum part
"Any constituent part of *the bacterial-type* flagellum, a 20 nm diameter filament composed of subunits of flagellin driven passively at its base by a motor powered by a transmembrane ion potential, typically a proton or sodium potential. Examples of this component are found in bacterial species." 

id: GO:0120100
name: bacterial-type flagellum motor
Should the space between H+ -driven and Na+ -driven be removed?

id: GO:0120101
name: bacterial-type flagellum stator complex
Should the space between H+ -driven and Na+ -driven be removed?

id: GO:0120107
name: bacterial-type flagellum rotor complex
definition: “*A part of the bacterial-type flagellum consisting* of a membrane-anchored ring and the motor switch complex, which participates in the conversion of proton/Na+ energy into the mechanical work of rotation and controls the direction of flagellar rotation." 

Thanks again!

**Comment by krchristie** [comment]
@paolaroncaglia 
1. Thanks for pointing out that it would be good to include the phrase "bacterial-type" to qualify the type of flagellum being described in these terms. I have done that for ALL of the child terms of "bacterial-type flagellum part", including ones we had not previously worked on. For terms that included either version of this sentence, which seemed redundant once the phrase "bacterial-type" was added, I have removed it:
- Examples of this component are found in bacteria.
- Examples of this component are found in bacterial species. 

I also added the GOC:cilia def dbxref to every definition I edited.


2. Regarding your question:
> Should the space between H+ -driven and Na+ -driven be removed?

I thought it would be worse to remove the space such that the '+' and the '-' were adjacent. Instead, I just removed the dash.

Here is the colorized diff to see everything I did:
https://github.com/geneontology/go-ontology/pull/14377/files
